### PR TITLE
DEV-1052 Add marc 008 field values to CRMS 

### PR DESCRIPTION
**Pull request by moseshll**
Reviewers: on the Perl side I am mainly interested in DRY violations in `Field008Formatter.pm` and other opportunities to make the code cleaner. On the CSS side, I'm just interested in avoiding anything truly yucky since I'm hopeless and it's all copy-pasta. Can be seen in action at

https://dev-2.babel.hathitrust.org/crms/cgi/crms?p=track&query=aeu.ark%3A%2F13960%2Ft14n0rn9t

(may need to re-paste URL after Shibboleth).

(Can refer to screenshots and description on https://hathitrust.atlassian.net/browse/DEV-1052)

- Add `lib/CRMS/Field008Formatter.pm` to generate HTML with breakdown of 008 fields.
  - Add `f008`  and `f008-annotation` classes to global CSS.
  - Add tests.
- Add display to _Query Bibliographic Rights_ and _Track Volumes_ page templates.
  - Make `Metadata` object (`$record`) available in the data structures consumed by these pages.
  - Add `Field008Formatter` hook in `CRMS.pm` hook for templates to use.
- Minor cleanup, specifically removing spurious `1;` final line from some test files (since they are not modules). Some readability tweaks as well.

### CODE REVIEW

File: web/css/review.css [hunk 1]
```diff
@@ -8,6 +8,27 @@ code {
...
+  margin: 4px;
+  letter-spacing: .2rem;
+  text-align: center;
+  display: inline-flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.f008 .f008-annotation {
+  margin-bottom: 0.1em;
```
Comment (carylwyatt) [inline web/css/review.css]: using `bottom` and `left` are totally fine and work! i've been trying to use [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) instead (for eventual i18n efforts) when I add/update styles:

`margin-bottom` => `margin-block-end`
`text-align: left` => `text-align: start`

not a necessary change, just thought you'd like to know!

the styling all looks good 👍

**Review by carylwyatt — APPROVED** [review]

File: cgi/CRMS.pm [hunk 2]
```diff
@@ -8169,7 +8171,14 @@ sub BibRights {
...
+}
+
+sub Field008Formatter {
+  my $self = shift;
+
+  use CRMS::Field008Formatter;
+  CRMS::Field008Formatter->new;
 }
 
 1;
```
Comment (mwarin) [inline cgi/CRMS.pm:8184]: If we can keep this class below 10K lines that would be great.

**Review by mwarin — APPROVED** [review]
It might not be the time or place, but I will keep nitpicking that we should change "docker-compose" to "docker compose" until I stop seeing it.

Builds according to the README, tests pass, etc.

Beautifully laid out in https://hathitrust.atlassian.net/browse/DEV-1052 what the problem is and what the goal is, and I think the proposed solution is just right. 

I see no issues with `Field008Formatter.pm`. Any attempt at making it "more generic" or "more perl-y" will just make it slower (by a yottasec), harder to read. 

No remarks on the CSS.

APPROVE.

File: web/css/review.css (hunk 1 above)
Comment (moseshll) [reply web/css/review.css]: There were a couple more to do in the main CSS file, I'll leave a couple of stragglers elsewhere in the repo on the to-do list for an eventual CSS cleanup.

File: cgi/CRMS.pm (hunk 2 above)
Comment (moseshll) [reply cgi/CRMS.pm:8184]: Should be following a shrinking trend as long as the cleanup tickets keep making it into sprints.

**Comment by moseshll** [comment]
Addressing the docker-compose -- I checked the other branch and I hadn't done it there yet.

**Pull request merged**